### DISCUSSION TITLE
How to use API reference

### DISCUSSION THREAD

**Discussion by praajna-shettigar** [Budibase/budibase#8604] [Help]
Hi

I have two table "Project" and "Task". with 1:N relationship
Once I create "Project" record it should save project record and create instance for "Task" table.
Relationship should match perfectly.
project field in the "Task" table should match with the "Project" created.

Want to achive this usecase using API reference. but I don't know how to use api reference.. need some help to achive this goal

Thanks,
Praajna

**Comment by melohagan** [comment]
Hey @praajna-shettigar 

By API reference, do you mean the public API? https://docs.budibase.com/reference/post_tables

Is there any reason you couldn't use a form? E.g.

- https://docs.budibase.com/docs/form-steps
- https://docs.budibase.com/docs/chaining-actions#example-creating-and-linking-two-table-entries-in-one-form
